hoomd/Integrator.h

```diff
@@ -32,10 +32,10 @@
     property that they have a time step, \a deltaT.
 
     Any number of ForceComputes can be used to specify the net force
-    for use with this integrator. They are added via calling
-    addForceCompute(). Any number of forces can be added in this way.
+    for use with this integrator. They are added through m_forces accessed via getForces. Any number
+    of forces can be added in this way.
 
-    All forces added via addForceCompute() are computed independently and then totaled up to
+    All forces added to m_forces are computed independently and then totaled up to
    calculate the net force and energy on each particle. Constraint forces (ForceConstraint) are
    unique in that they need to be computed \b after the net forces is already available. To
    implement this behavior, call addForceConstraint() to add any number of constraint forces. All
```

Comment:
Another instance of `addForceConstraint()` that needs to be removed from the docstring. This paragraph also seems to duplicate the previous paragraph.